lib/cereal/grpc/grpc.go

```diff
@@ -170,21 +270,32 @@ func (g *GrpcBackend) DequeueWorkflow(ctx context.Context, workflowNames []strin
 		return nil, nil, err
 	}
 
-	resp, err := s.Recv()
-	if err != nil {
-		cancel()
-		if st, ok := status.FromError(err); ok {
-			if st.Code() == codes.NotFound {
-				return nil, nil, cereal.ErrNoWorkflowInstances
+	var blob []byte
```

Comment:
can we do this recursive receive work in a function, for better readability of this logic.

Reply:
Made the change, thanks for the suggestion😊